docker/1.4.4/config/README.md

```diff
@@ -0,0 +1,4 @@
+# Configuration
+
+This directory is mounted as a volume for the docker container running Mosquitto.
+
```

Comment:
Possibly here you could describe what you have changed in the configuration from the mosquitto defaults?